schemas/json/theme.json

```diff
@@ -219,6 +219,11 @@
 							"type": "boolean",
 							"default": false
 						},
+						"width": {
```

Comment:
one other note: the search block's width side panel still appeared, even when spacing.width is set to false in theme.json (I also added the following to the search block); not sure if that is an issue. 

```
"spacing": {
			"width": true
		},
```

Reply:
Thanks for taking the time to look over this!

I wanted to keep the scope of this PR narrow and contained to the Button block; I actually wasn't aware that the Search block had width settings! With the new `theme.json` property that this PR adds, we could look into adding support to the Search block in a follow-up feature. That way we can keep the testing steps and branch scope concise.

What do you think?

Reply:
I am personally fine with either; I think making an issue that states width support is missing for the search block would be helpful without being too burdensome? :)

Reply:
@skorasaurus issue has been created! #42159 😄 Let me know if there's anything else I need to do here, thanks!

Reply:
I believe the default needs to be true, because this setting has been available to users and they may expect it to be available.
Personally, I would disable this option for _some_ projects, and I welcome the possibility to do so, but I think that because it is an existing option, disabling it needs to be the exception.

Reply:
@carolinan I agree, we definitely don't want this panel disappearing for people who expect it to be there and aren't staying on top of release notes. I've made some adjustments so that we expressly look for `spacing.width === false` when determining whether or not to display the panel.